### ISSUE TITLE
As of July 2023, users must provide an api key and key id to create an access token.

### ISSUE THREAD

**Issue by deltabrav0** [RMCob/homebridge-wyze-robovac#17]
**Describe The Bug:**
Unable to connect to wyze.com

**To Reproduce:**
start home bridge, first connection to wyze. getVaccuumDeviceList method.

**Expected behavior:**
Connection and authentication successful.

**Logs:**

```
Traceback (most recent call last):
  File "/usr/local/lib/node_modules/homebridge-wyze-robovac/py_helpers/getVacuumDeviceList.py", line 13, in <module>
    client = Client(email=sys.argv[1], password=os.sys.argv[2])
  File "/Users/dbutler/Library/Python/3.9/lib/python/site-packages/wyze_sdk/api/client.py", line 72, in __init__
    self.login()
  File "/Users/dbutler/Library/Python/3.9/lib/python/site-packages/wyze_sdk/api/client.py", line 176, in login
    raise WyzeClientConfigurationError(
wyze_sdk.errors.WyzeClientConfigurationError: Must provide a Wyze API key and id.

As of July 2023, users must provide an api key and key id to create an access token. For more information, please visit https://support.wyze.com/hc/en-us/articles/16129834216731.
```

**Plugin Config:**

```json
{
    "name": "WyzeRoboVac",
    "username": "xxxxxx",
    "password": "xxxxxxxx",
    "mfaCode": "781419",
    "statusCheckRefreshInterval": 30,
    "idleBatteryCheckRefreshInterval": 1800,
    "busyBatteryCheckRefreshInterval": 120,
    "path2py_stubs": "/usr/local/lib/node_modules/homebridge-wyze-robovac/py_helpers",
    "debugLevel": 0,
    "platform": "WyzeRoboVac"
}
```

**Screenshots:**

**Environment:**

* **Plugin Version**:  homebridge-wyze-robovac v1.2.1
* **Homebridge Version**: v1.6.1
* **Node.js Version**: v16.16.0
* **NPM Version**: 8.11.0
* **Operating System**:  macOS ->

**Comment by sajalkherde** [comment]
@deltabrav0 @RMCob can I get your thoughts on this? this will satisfy the api key and key id requirement as per the new Wyze changes. I am very new to this so any recommendations appreciated!

**Comment by RMCob** [comment]
This problem is resolved with release 1.4.0
